Plotting_Scripts/CCP_plottingroutines.py

```diff
@@ -692,13 +687,13 @@ def plot_crosssection_any(self, lon1,lon2,lat1,lat2,numpoints=200,amplify=1.,nam
                     crossec[i, j] = 1000.
 
         # Map
-        plt.subplot(2, 2, 2)
-        
-        m = plt.axes(projection=ccrs.Mercator())
+        m = plt.subplot(2, 2, 2, projection=ccrs.Mercator())
```

Comment:
There is use of both Mercator and Plate Carree projections throughout. Is this on purpose?